testData/dbData/views.json

```diff
@@ -11,6 +11,18 @@
             "findAuthorizationByAccessToken": {
                 "map": "function(doc) {if (doc.type === 'gpiiAppInstallationAuthorization' && doc.revoked === false) {emit(doc.accessToken, {'_id': doc.clientId, 'authorization': doc})}}"
             },
+            "findKeysForPrefsSafe": {
+                "map": "function(doc) {if (doc.type === 'gpiiKey' || doc.type === 'gpiiCloudSafeCredential') emit(doc.prefsSafeId, null)}"
+            },
+            "listPrefsSafes": {
+                "map": "function(doc) {if (doc.type === 'prefsSafe') emit(doc._id, {'name': doc.name, 'email': doc.email, 'created': doc.timestampCreated, 'updated': doc.timestampUpdated})}"
+            },
+            "findPrefsSafeByName": {
```

Comment:
This view doesn't seem to be used.

Reply:
Thanks for catching this, it was from earlier in the gpii express integration work, and did not end up being used.  Removed.